### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by selgamal**
#### Reason for change
#385 
Proposed update to usage options, add pip installation and reference to github releases 

#### Description of change
579e8eea1b1d03ff4d4acda2bbe285eca94d5d60 Adds installation instruction using pip, and installation for development version to INSTALLATION.md, also adds absloute links for installation, contribution and license to README.md (relative links don't seem to work on PyPi description page.

421b864e4018c7a624d044a7601015048cebfc55 add table of contents to README.md

e22820b7935ddafa5c29395e0fdc2226241420f2 Merges INSTALLATION.md into README.md so installation instructions are visibile directly from README page.

29c9c5fae3a029f8ddbe051e169e4a0b1cabd3ef add tests status badges to read me

#### Steps to Test
Changes to README.md file, just needs to be reviewed.

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by sagesmith-wf — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 <div align="center">
   <img src="http://arelle.org/arelle/wp-content/themes/platform/images/logo-platform.png">
 </div>
 
+- [Arelle](#arelle)
+  - [Features](#features)
+- [Installation](#installation)
+  - [Install PyPi package](#install-pypi-package)
```
Comment (austinmatherne-wk) [inline README.md:9]: nit: the "i" in "PyPI" is capitalized:
```suggestion
  - [Install PyPI package](#install-pypi-package)
```

File: README.md [hunk 2]
```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
 * QuickBooks is supported by XBRL-GL.
 
 
-## Installation
+# Installation
 
-Please check out our [Arelle install guide](INSTALLATION.md).
+The implementation is in Python 3.10, and is intended for Windows (any recent),
+Mac OS-X 10.9+, Unix or Linux. The standard installation includes both desktop GUI,
+a localhost RESTful web server and CLI.
```
Comment (austinmatherne-wk) [inline README.md:44]: * We don't officially support Python 3.10 yet. Hopefully within the next week that'll change.
* Preexisting from the installation guide - supported OS versions are out of date

```suggestion
The implementation is in Python 3.9 and is intended for Windows, macOS, or Linux (tested against Ubuntu and RHEL). The standard desktop installation includes a GUI, a RESTful web server, and CLI. We try to support all operating system versions that still receive security updates from their development teams.
```

File: README.md [hunk 3]
```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
...
 
+## Install PyPi package
+```shell
+pip install arelle-release
+```
 
-## Contribution guidelines
+## Install development version from github
+```shell
+pip install git+https://git@github.com/arelle/arelle.git@master#egg=arelle-release
```
Comment (austinmatherne-wk) [inline README.md:53]: I don't think `#egg=arelle-release` is needed?
```suggestion
pip install git+https://git@github.com/arelle/arelle.git@master
```

File: README.md [hunk 4]
```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
...
+# Installation
 
-Please check out our [Arelle install guide](INSTALLATION.md).
+The implementation is in Python 3.10, and is intended for Windows (any recent),
+Mac OS-X 10.9+, Unix or Linux. The standard installation includes both desktop GUI,
+a localhost RESTful web server and CLI.
 
+## Install PyPi package
+```shell
+pip install arelle-release
```
Comment (austinmatherne-wk) [inline README.md:48]: We should document [supported optional extras](https://github.com/Arelle/Arelle/blob/bb7eb53e5d26adbd7fb307d9bb4b1c7a0b5b592d/pyproject.toml#L35-L55):
```suggestion
The Arelle python package defines optional extra dependencies for various plugins and use cases.
* Crypto (security plugin dependencies)
* DB (database plugin dependencies)
* EFM (EdgarRenderer plugin dependencies - does not include the EdgarRenderer, just the dependencies required to run it)
* ObjectMaker (ObjectMaker plugin dependencies)
* WebServer (dependencies for running the Arelle web server)
```shell
pip install arelle-release
# or for all extra dependencies
pip install arelle-release[Crypto,DB,EFM,ObjectMaker,WebServer]
```

File: README.md [hunk 5]
```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
...
-If you want to contribute to Arelle, be sure to review the
-[contribution guidelines](CONTRIBUTING.md).
+## Install distributions
+Distributions include Python version and resources needed to run Arelle.
+
+Arelle provides distributions for the following operating systems:
+* Windows (x64 and x32 architecture)
+* Mac OS
+* Linux (Ubuntu)
+* Red Hat
```
Comment (austinmatherne-wk) [inline README.md:63]: Preexisting, this should be tweaked:
```suggestion
* Windows
* macOS (Intel)
* Linux (Ubuntu and Red Hat Enterprise Linux)
```

File: README.md [hunk 6]
```diff
@@ -27,20 +37,48 @@ be integrated with other applications and languages utilizing its web service.
...
+
+Distributions can be downloaded from:
+* [Arelle website](https://arelle.org/arelle/pub/)
+* [github releases](https://github.com/Arelle/Arelle/releases)
```
Comment (austinmatherne-wk) [inline README.md:67]: nit:
```suggestion
* [GitHub releases](https://github.com/Arelle/Arelle/releases)
```

File: README.md [hunk 7]
```diff
@@ -1,9 +1,19 @@
+[![Run Python Unit Tests](https://github.com/Arelle/Arelle/actions/workflows/tests.yml/badge.svg)](https://github.com/Arelle/Arelle/actions/workflows/tests.yml)  [![Run Xbrl Conformance Suites](https://github.com/Arelle/Arelle/actions/workflows/conformance-suites.yml/badge.svg)](https://github.com/Arelle/Arelle/actions/workflows/conformance-suites.yml)  [![Build and publish python package](https://github.com/Arelle/Arelle/actions/workflows/python-package.yml/badge.svg)](https://github.com/Arelle/Arelle/actions/workflows/python-package.yml)  [![Build for Linux](https://github.com/Arelle/Arelle/actions/workflows/build-linux.yml/badge.svg)](https://github.com/Arelle/Arelle/actions/workflows/build-linux.yml)  [![Build for Mac](https://github.com/Arelle/Arelle/actions/workflows/build-mac.yml/badge.svg)](https://github.com/Arelle/Arelle/actions/workflows/build-mac.yml)  [![Build for Windows](https://github.com/Arelle/Arelle/actions/workflows/build-windows.yml/badge.svg)](https://github.com/Arelle/Arelle/actions/workflows/build-windows.yml)
```
[the remark this answers is not in the document]
Comment (austinmatherne-wk) [reply README.md:1]: I believe they're all running against the default branch (master), so the status is misleading for most of them. Only the python package and OS builds will update (when we tag a new release). I think we should leave badges out of this PR and consider them in a separate [discussion](#294).

As an aside, maybe we should run all of the test suites on the release workflow (possibly even include the long running EFM suite) before running the builds?

[the remark this answers is not in the document]
Comment (selgamal) [reply README.md:1]: So we are removing all badges for now?

[the remark this answers is not in the document]
Comment (austinmatherne-wk) [reply README.md:1]: Yes. Looks like Derek just opened a PR for pypi and python versions, but let's remove the rest until we update the release pipelines to also run the test suites.

**Review by sagesmith-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**